addons/core/translations/en-us.yaml

```diff
@@ -32,6 +32,7 @@ titles:
   no-results: No Results
   choose-a-provider: Choose a provider
   provider: Provider
+  sync-interval: Sync Interval
```

Comment:
I have no context on how we are organizing translations, perhaps @gsusmi can provide some light.

But as per my understanding (I can be wrong as mentioned above) this file is for root translations, and the content of the translations added here looks to me a form field, Shouldn't they live within `/form`?

thoughts?

Reply:
I think that is what @randallmorey suggested when I added the `Latest IP Addresses` label and field to the host form. I agree it and the description should be moved into `/form`.